modules/batch/src/main/java/com/devonfw/module/batch/common/base/SpringBootBatchCommandLine.java

```diff
@@ -97,29 +141,25 @@ public static void main(String[] args) throws Exception {
       op = Operation.STOP;
     } else {
 
-      for (int i = 3; i < args.length; i++) {
+      for (int i = 2; i < args.length; i++) {
 
         parameters.add(args[i]);
       }
     }
 
-    new SpringBootBatchCommandLine().execute(op, configurations, args[2], parameters);
+    new SpringBootBatchCommandLine().execute(op, args[0], args[1], parameters);
   }
 
   private static void handleIncorrectParameters() {
 
     LOG.error("Incorrect parameters.");
     LOG.info("Usage:");
-    LOG.info("java com.devonfw.module.batch.common.base.SpringBootBatchCommandLine"
-        + " <SpringBootConfiguration> <BatchJobConfiguration>" + " <JobName> param1=value1 param2=value2 ...");
+    LOG.info(
+        "java -jar my-app-batch-bootified.jar <SpringBootConfiguration> <JobName> param1=value1 param2=value2 ...");
```

Comment:
Just a thought in this:
If we deprecated this anyhow, does it make sense to change the CLI API after all?
Maybe it is even more useless without that change. So just a note to consider or ignore.

Reply:
you're right. i'll change that.

Reply:
I rivisited this topic.
The problem is that the release implementation the launcher in devonfw will not work together with spring boot. The idea for batches in devon was always to reuse as much as possible of your application code for implementing your batches. We provided solution which accomplished that in the first release of devon4j-batch.
When introduction spring boot and java based configs this solution broke and there was never a real solution how to start your batch in the "context" of your boot app, how to handle classpath when having a bootified jar and so on.
So the current change in this PR makes things working again with the devon batch launcher and spring boot apps. Since this is not an optimal solution (the launcher is not required), I fixed and deprecated it.
We could leave the implementation as is, but we cannot document how to use it (since it is not working ;-))